bunsen/bunsen-avro/src/main/java/com/cerner/bunsen/avro/converters/DefinitionToAvroVisitor.java

```diff
@@ -37,7 +37,6 @@
 import org.apache.avro.JsonProperties;
 import org.apache.avro.Schema;
 import org.apache.avro.Schema.Field;
-import org.apache.avro.Schema.Type;
```

Comment:
Are these `Schema.Type` related changes required? If not, can we revert these to make the PR more focused?

Reply:
Okay, it would be nice to do that for `Schema.Field` too for consistency, but up to you.

Reply:
Done